### PR TITLE
Don't extend multi answers beyond page width.

### PR DESCRIPTION
**Pull request by islemaster**
Fixes [Pivotal #100982938](https://www.pivotaltracker.com/story/show/100982938).
Use `calc()` to remove left margin on answer div from its 100% width.  Prevents page from scrolling horizontally (which is apparently only an issue in IE9+10).

Before:
screenshot from 2015-08-17 16 55 30

After:
screenshot from 2015-08-17 16 54 33

### CODE REVIEW

File: dashboard/app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -1685,6 +1686,9 @@ $information-desktop: "only screen and (min-width: 1024px)";
     @media #{$information-desktop} {
       width: 55%;
     }
+    &.question-content-blank {
+      width: calc(100% - #{$multi-left-margin});
```
Comment (breville) [inline dashboard/app/assets/stylesheets/application.css.scss]: wow, i did not know this was possible

**Comment by breville** [comment]
if it works on all our browsers, then lgtm :)

**Comment by islemaster** [comment]
:P I'd better figure out how to write a UI test to prove that this level doesn't scroll horizontally.

**Comment by islemaster** [comment]
Added UI test to check this on all browsers (it was even an issue in latest Chrome, I discovered).

So close!
screenshot from 2015-08-17 17 35 28

That failing iPhone5S test actually runs on iPhone6 on Browserstack, and appears to have all sorts of stuff going wrong... Notice that it didn't rotate to landscape, and didn't return ANY value for the JS expression it's trying to execute.

screenshot from 2015-08-17 17 36 10

**Comment by islemaster** [comment]
A simple re-run corrected the rotation issue, but the expression still returns nothing (which reads as "false" to the test).  This works on other iOS devices.  I'm not sure what's up in this case.  Will do some manual testing.

screenshot from 2015-08-17 17 42 23

**Comment by islemaster** [comment]
Manual testing on Browserstack shows no issue on iPhone 6 Safari.  I can't hook up to the Safari debugger on Browserstack (that I can tell) so I can't diagnose why the UI tests aren't getting a valid response.  I'm disabling the new test for mobile until we can solve that issue (might require someone on OSX taking a look).

@breville if that's okay with you, I'll go ahead and merge the fix.

**Pull request merged**